src/main/java/core/basesyntax/RightTriangleSupplier.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class RightTriangleSupplier {
+    private Random random = new Random();
+    private RandomSize size = new RandomSize();

```

Comment:
The name of the 'size' object is not clear, consider renaming it to 'randomSize' to reflect its purpose.